internal/cmd/groups.go

```diff
@@ -107,3 +109,70 @@ func newGroupsListCmd(cli *CLI) *cobra.Command {
 		},
 	}
 }
+
+func newGroupsAddCmd(cli *CLI) *cobra.Command {
+	return &cobra.Command{
+		Use:   "add GROUP",
+		Short: "Create a group",
+		Args:  ExactArgs(1),
+		Example: `# Create a group
+$ infra groups add Engineering`,
```

Comment:
Will group names be case sensitive? The capital 'E' in 'Engineering' implies that possibility to me.